packages/bpk-react-utils/src/BpkDialogWrapper/BpkDialogWrapper.tsx

```diff
@@ -177,6 +182,6 @@ export const BpkDialogWrapper = (props: Props) => {
           </div>
         </dialog>
       </CSSTransition>
-    </div>
+    </div >
```

Comment:
Is this an automatic linting change?

Reply:
Not sure what happened there - the linter is happy with or without the space so will remove 😆